PatientWebApplication/UserMicroserviceApi/Model/SecretaryUser.cs

```diff
@@ -0,0 +1,30 @@
+/***********************************************************************
+ * Module:  Secretary.cs
+ * Author:  Luna
+ * Purpose: Definition of the Class Secretary.Secretary
+ ***********************************************************************/
+
+using System;
+
+namespace UserMicroserviceApi.Model
+{
+    public class SecretaryUser : EmployeeUser
+    {
+        public string room { get; set; }
```

Comment:
Change property name to start with upper letter and apply in rest of code (remove 'this' from constructors).